plugins/ltac/tacinterp.ml

```diff
@@ -890,10 +890,9 @@ let interp_intro_pattern_naming_option ist env sigma = function
 let interp_or_and_intro_pattern_option ist env sigma = function
   | None -> sigma, None
   | Some (ArgVar (loc,id)) ->
-      (match coerce_to_intro_pattern env sigma (Id.Map.find id ist.lfun) with
-      | IntroAction (IntroOrAndPattern l) -> sigma, Some (loc,l)
-      | _ ->
-        user_err ?loc (str "Cannot coerce to a disjunctive/conjunctive pattern."))
+    (match interp_intro_pattern_var loc ist env sigma id with
```

Comment:
This could be formatted so the only actual difference, the match discriminee, gets marked by Github.

Reply:
Good point, done (this also caught that I accidentally dropped a period in the error message).